apps/fabric-website-resources/webpack.serve.config.js

```diff
@@ -21,6 +21,7 @@ module.exports = resources.createServeConfig({
       'office-ui-fabric-react/lib': path.resolve(__dirname, '../../packages/office-ui-fabric-react/src'),
       '@uifabric/fluent-theme$': path.join(__dirname, '../../packages/fluent-theme/src'),
       '@uifabric/theme-samples$': path.join(__dirname, '../../packages/theme-samples/src'),
+      '@uifabric/azure-themes$': path.join(__dirname, '../../packages/azure-themes/src'),
```

Comment:
There is also [an experiments version of this file](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/experiments/webpack.serve.config.js) if you want to add it there.